RPGLootFeed/Features/ItemLoot/ItemLoot.lua

```diff
@@ -393,8 +393,38 @@ function ItemLoot.Element:new(info, quantity, fromLink)
 						G_RLF.tertiaryToString[TertiaryStats.Indestructible]
 					)
 				end
-				return secondaryText
 			end
+			
+			local alwaysShowSubTypes = { Cloth = true, Leather = true, Mail = true, Plate = true }
+			local equipLocShowSubType = { INVTYPE_WEAPONMAINHAND = true, INVTYPE_WEAPONOFFHAND = true, };
+			local equipLocShowOnlySubType = { INVTYPE_WEAPON = true, INVTYPE_2HWEAPON = true }
+			local extraText = ""
+			if info.itemEquipLoc and info.itemSubType and alwaysShowSubTypes[info.itemSubType] then
+				extraText = " [" .. _G[info.itemEquipLoc] .. " - " .. info.itemSubType .. "]"
+			elseif info.itemEquipLoc and equipLocShowSubType[info.itemEquipLoc] and info.itemSubType then 
+				extraText = " [" .. _G[info.itemEquipLoc] .. " - " .. info.itemSubType .. "]"
+			elseif info.itemEquipLoc and equipLocShowOnlySubType[info.itemEquipLoc] and info.itemSubType then 
+				extraText = " [" .. info.itemSubType .. "]"
+			elseif info.itemEquipLoc then
+				extraText = " [" .. _G[info.itemEquipLoc] .. "]"
+			end
+			if extraText then
```

Comment:
Pretty sure we want this one to be an explicit check for empty string unless we start with `extraText = nil`

<img width="265" height="625" alt="image" src="https://github.com/user-attachments/assets/8b5e59a8-04ee-44d5-bc06-f14c7c2ec1c1" />

Lua truthy/falsey is different than most programming languages when it comes to things like this.

```suggestion
			if extraText ~= "" then
```

Is that correct based on the intention here?

Although, admittedly not a big deal because it would just be appending an empty string (sometimes with color 😛 )